person.rb

```diff
@@ -1,19 +1,33 @@
-class Person
-  attr_accessor :name, :age
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
-    @id = Randaom.rand(1..1000)
```

Comment:
- [ ] Please kindly take note that I was not able to create a teacher and a student, I got the error message screenshot below 👇. Kindly resolve it. 🙏 

<img width="770" alt="Untitled" src="https://github.com/HossainAraf/school-library/assets/103386183/56b0ec17-4d45-4af7-9489-35330abbf53e">